tools/microtvm/vm/kitware-archive-latest.asc

```diff
@@ -0,0 +1,81 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
consider move to apps as we use that folder for conventional misc tools

Reply:
my thinking is that this is more like a `docker` directory than an `apps` subdir. happy to move wherever, though. what do you think?

Reply:
Let us still move to apps to reduce the amount of top-level directories.